opm/material/fluidmatrixinteractions/EclEpsTwoPhaseLaw.hpp

```diff
@@ -482,7 +482,15 @@ class EclEpsTwoPhaseLaw : public EffLawT::Traits
             return unscaledPcnw*alpha;
         }
         else if (params.config().enablePcScaling()) {
-            Scalar alpha = params.scaledPoints().maxPcnw()/params.unscaledPoints().maxPcnw();
+            const auto& scaled_maxPcnw = params.scaledPoints().maxPcnw();
+            const auto& unscaled_maxPcnw = params.unscaledPoints().maxPcnw();
+
+            Scalar alpha;
+            if (scaled_maxPcnw == unscaled_maxPcnw)
+                alpha = 1.0;
```

Comment:
Can we have a situation where unscaled is zero and scaled is not zero or is that not acceptable input?

Reply:
I don't know enough about this; my attempt was to "fix" this as non-invasive as possible. The situation you describe could/should be handled with an additonal:

```C++
if (!std::isfinite(alpha))
     throw std::logic_error("Invalid numerical value encountered ....");
```

happy to add that. But as I have said numerous times - I am careful in this part of town.